c2corg_ui/templates/xreport/view.html

```diff
@@ -0,0 +1,194 @@
+  show_archive_data, show_areas, show_locale_title, get_activities, show_maps,
+  show_associated_waypoints, show_associated_outings, show_associated_routes, show_associated_xreports,
+  show_associated_articles, show_float_buttons,
+  delete_association_confirmation_modal, associated_waypoints_featurelist,
+  associated_images_featurelist, show_badge, get_licence"/>
+<%namespace file="helpers/view.html" import="show_fulldate, get_xreport_general, get_xreport_participants, get_xreport_personal"/>
+
+<%
+xreport_id = xreport['document_id']
+xreport['doctype'] = 'xreports'
+other_langs, missing_langs = get_lang_lists(xreport, lang)
+## geometry = xreport['geometry']
+## geometry4326 = geometry
+%>
+
+<%block name="pagelang">lang="${lang}"</%block>
+
+<%block name="pagetitle"><title>${show_locale_title(locale)}</title></%block>
+
+<%block name="metarobots">
+  % if version:
+    <meta name="robots" content="noindex,follow">
+  % else:
+    <meta name="robots" content="index,follow">
+  % endif
+</%block>
+
+<%block name="moduleConstantsValues">
+  <%
+      if xreport['geometry']:
+          geometry = xreport['geometry']['geom_detail'] \
+              if xreport['geometry']['geom_detail'] else xreport['geometry']['geom']
+      else:
+          geometry = None
+  %>
+
+  module.value('mapFeatureCollection',
+  % if geometry:
+    {
+      "type": "FeatureCollection",
+      "properties": {},
+      "features": [
+      {
+        "type": "Feature",
+        "geometry": ${geometry | n},
+        "properties": {
+          "title": ${dumps(locale['title']) | n},
+          "lang": "${locale['lang']}",
+          "documentId": ${xreport['document_id']},
+          % if xreport['elevation'] is not None:
+            "elevation": ${xreport['elevation']},
+          % endif
+          "module": "xreports",
+          "highlight": false
+        }
+      }
+    ]}
+  % else:
+    null
+  % endif
+  );
+
+  module.value('documentData', {
+    "document_id": ${xreport.get('document_id')},
+    "lang": "${lang}",
+    "type": "x",
+    "topic_id": ${dumps(locale.get('topic_id'))}
+    % if not version:
+       , "associations": ${dumps(xreport.get('associations')) | n}
+    % endif
+  });
+</%block>
+
+<header class="view-details-title">
+  <h1 class="routes">
+    <div class="icons-header routes">${get_activities(xreport, 'top')}</div>
+    <span class="title">${show_locale_title(locale)}</span>
+    ${show_badge(xreport, 'xreport')}
+  </h1>
+  <div class="outing-date" ng-cloak>${show_fulldate(xreport['date'])}</div>
+</header>
+
+% if xreport.get('areas'):
+  <article class="location-static text-center">
+    ${show_areas(xreport)}
+    % if xreport.get('elevation'):
+      <p class="text-center"><b>${xreport['elevation']} m</b></p>
+    % endif
+##     TODO SHOW COORDINATES ON THE YELLOW PAPER IN THE HEADER MAP
+##     % if geometry4326:
+##       <p>${round(geometry4326.x)} °E / ${round(geometry4326.y)} °N</p>
+##     % endif
```

Comment:
@tsauerwein has fix the reprojection problems for waypoints. See https://github.com/c2corg/v6_ui/pull/1085
You can probably do the same for xreports.
OK to do it after golive if not enough time.